Applications/AlexNet/README.md

```diff
@@ -0,0 +1,25 @@
+### How to run
+1. you should compile nntrainer with this page --> [Getting Started](https://github.com/nnstreamer/nntrainer/blob/main/docs/getting-started.md)
+
+2. you need to make Dataset(.dat file) with this pahe --> [Datagen](https://github.com/nnstreamer/nntrainer/blob/main/Applications/utils/datagen/cifar/How-to-gen-data.md)
+
+Once you compile the codes, you can run with
+``` bash
+$ cd ${nntrainer_dir}
+$ meson ${build_dir}
+$ cd ${build_dir}
+$ ninja
+$ sudo ninja install
+$ ./Applications/AlexNet/jni/nntrainer_alex \
+../Applications/AlexNet/res/alex.ini ../Applications/AlexNet/res/
```

Comment:
We designate a res directory in ${build_dir} to storing the resource files for application.
Instead of directly using the resource files in source directory how about using this directory?
Please refer meson.build in Applications/TransferLearning/Draw_Classification/jni/meson.build:4

Reply:
FIX
``` meson
nntr_alex_resdir = nntr_app_resdir / 'AlexNet'
run_command('cp', '-lr', res_path, nntr_alex_resdir)
```
But ```.dat``` files almost 150MB so it comment as below

``` bash
$ cd ${nntrainer_dir}
$ meson ${build_dir}
$ cd ${build_dir}
$ ninja
$ sudo ninja install
$ ./Applications/AlexNet/jni/nntrainer_alex ./res/app/AlexNet/alex.ini {.dat files dir}
```